docs/BUILDING.md

```diff
@@ -75,9 +75,71 @@ Using `rpi-imager`, install the Ubuntu 20.04.1 LTS (Raspberry Pi 3/4) 64-bit
 server OS for arm64 architectures on a micro SD card.
 
 Boot the SD card, login with the default user account "ubuntu" and password
-"ubuntu", then proceed with "How to install prerequisites on Linux".
+"ubuntu", then we can configure the wireless network to connect your Raspberry
+Pi 3/4 to the internet and using it remotely.
 
-Finally, install some Raspberry Pi specific dependencies:
+Getting setup with Wi-Fi:
+
+Please DON'T configure your wireless network using initial configuration files
+that will be loaded during the first boot.
+
+1. Create a WiFi configuration file wpa_supplicant.conf under directory
+   /etc/wpa_supplicant:
+
+```
+Open the wpa-supplicant configuration file in vim:
+
+sudo vim /etc/wpa_supplicant/wpa_supplicant.conf
+
+Add the following network configuration:
+
+    ctrl_interface=/run/wpa_supplicant
+
+    network={
```

Comment:
Aren't we supposed to set this config during network provisioning?

Reply:
Yes, the question is how to save the configuration throughout the power cycle without using the wpa_supplicant config file.
This is up to design, CHIP can maintain the  config by itself and apply to wpa_supplicant on each restart, let's have more discussion on which route we want to proceed offline.